source/_integrations/light.markdown

```diff
@@ -25,7 +25,7 @@ Most lights do not support all attributes. You can check the integration documen
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
 | `entity_id` | no | String or list of strings that point at `entity_id`s of lights. To target all the lights use all as `entity_id`.
-| `transition` | yes | Number that represents the time (in seconds) the light should take to transition to the new state.
+| `transition` | yes | Number that represents the time (in seconds) the light should take to transition to the new state. Only integers are allowed.
```

Comment:
That is actually incorrect. It is a float.

Reply:
Oh okay.

According to the discord, using anything beside integers is not a good idea:

```
Hypfer Today at 16:34
https://www.home-assistant.io/integrations/light/

can I use fractions of a second for transition?
Vasiley Today at 16:35
nope
Tinkerer Today at 16:36
You might be able to use them, but even if it doesn't error, it won't give you what you expect
Hypfe rToday at 16:37
okay thx
it might make sense to add that to the documentation
```

Reply:
That is nice, but that is not correct.

If you doubt my judgment, please look at the code yourself:

<https://github.com/home-assistant/core/blob/dev/homeassistant/components/light/__init__.py#L85>

Let me know if you think I was incorrect.

Reply:
If you found an integration that does have issues with processing a float, please make sure to open up an issue our GitHub, because that should be considered a bug.

Reply:
I see